setup.py

```diff
@@ -21,6 +21,8 @@
 ]
 
 extras_require = {
+    'plone4': [
+        'Products.TinyMCE'],
```

Comment:
Did you run into a problem without that?
Since TinyMCE is part of Plone 4 this should not be necessary.

Reply:
I did not try without. It was there already, I just moved it into an extra, because it wouldn't work in plone5.

Reply:
@busykoala ok then, let's just remove the extra

Reply:
Removed in https://github.com/4teamwork/ftw.book/pull/112/commits/46207158d9c3401176719ac7608951489140381e